src/Shared/PackageManagers/ProjectManagerFactory.cs

```diff
@@ -8,55 +8,153 @@ namespace Microsoft.CST.OpenSource.PackageManagers
     using System.Linq;
     using System.Net.Http;
 
-    public static class ProjectManagerFactory
+    public class ProjectManagerFactory
     {
         /// <summary>
-        /// Get an appropriate project manager for package given its PackageURL.
+        /// The delegate to use when defining the function to create a project manager with.
```

Comment:
```suggestion
        /// This delegate takes the path to operate a project manager in and returns a ProjectManager.
```

Add another line describing how you would actually use this by constructing a dictionary?

Reply:
I added an example and a pointer to the CreateDefaultManagers method